src/piton/extractors/omop_converter.py

```diff
@@ -224,57 +226,65 @@ def remove_pre_birth(patient: Patient) -> Optional[Patient]:
     return Patient(patient_id=patient.patient_id, events=new_events)
 
 
-# There is no point holding onto patients with just a single time point
-def remove_small_patients(patient: Patient) -> Optional[Patient]:
-    if len(set(event.start for event in patient.events)) <= 2:
+def _remove_small_patients(patient: Patient) -> Optional[Patient]:
```

Comment:
```suggestion
def _remove_small_patients(patient: Patient, min_num_events: int) -> Optional[Patient]:
```

Reply:
- Filtering should be a configurable choice (also -- isn't the minimum points 2? birth and 1 vist?) 
- Names should be precise. What does "small" mean? Use something more descriptive (`filter_by_len`) and should e parameterizable per @scottfleming

Reply:
Renamed to remove_short_patients with a config flag.

You generally need at least 3 dates for any non-trivial use.